File: objective-c/helloworld.m
// Hello World

// Compile with:
// $ gcc -framework Foundation -o bin/helloworld helloworld.m

// Run with:
// $ ./bin/helloworld

#import <Foundation/Foundation.h>

int main (int argc, const char * argv[])
{
	NSAutoreleasePool *pool = [[NSAutoreleasePool alloc] init];
	NSLog (@"hello, world");
	[pool drain];
	return 0;
}

File: erlang/helloworld.erl
% Hello World

% Compile with:
% $ erlc -o bin helloworld.erl

% Run with:
% $ cd bin; erl -noshell -s helloworld start -s init stop

-module(helloworld).
-export([start/0]).

start() ->
    io:fwrite("hello, world\n").
